test/manipulation.test.js

```diff
@@ -335,11 +334,17 @@ describe('manipulation', function() {
 
         Product.create({name: 'a-name'}, function(err, p) {
           if (err) return done(err);
-          Product.create({id: p.id, name: 'duplicate'}, function(err) {
-            if (!err) {
-              return done(new Error('Create should have rejected duplicate id.'));
+          Product.create({id: p.id, name: 'duplicate'}, function(err, result) {
+            if (isCassandraConnector) {
```

Comment:
create is implemented as INSERT.  Cassandra INSERT overwrites if exists.
Cassandra does suuport user-provided ids.